app/components/repository-sidebar.js

```diff
@@ -68,20 +70,18 @@ export default Ember.Component.extend({
     return runningAmount + queuedAmount;
   },
 
-  @computed('features.proVersion', 'jobState.runningJobs.[]')
-  runningJobs(proVersion, runningJobs) {
-    if (!proVersion) { return []; }
+  @computed('jobState.runningJobs.[]')
+  runningJobs(runningJobs) {
     return runningJobs;
   },
 
-  @computed('features.proVersion')
-  queuedJobs(proVersion) {
-    if (!proVersion) { return []; }
-
+  @computed('currentUser')
+  queuedJobs(user) {
     const queuedStates = ['created'];
     const result = this.get('store').filter(
       'job',
-      job => queuedStates.includes(job.get('state'))
+      job =>
+        queuedStates.includes(job.get('state') && user && user.hasAccessToRepo(job.get('repo')))
```

Comment:
Actually, I think that this won't be enough in some cases, because I think that we don't update `permissions` array when a new repository is added. So it will be OK when you have access to a repo when the page loads, because the repo is already included in permissions. If you, however, keep the page opened and a new repo is added in the meantime, it won't be in permissions, so `isCurrentUserACollaborator` will be false. This will be fixed by `@is_collaborator` property, but we will also need to add this to Pusher.